cpp/CMakeLists.txt

```diff
@@ -915,8 +915,8 @@ if(CUDF_LARGE_STRINGS_DISABLED)
   target_compile_definitions(cudf PRIVATE CUDF_LARGE_STRINGS_DISABLED)
 endif()
 
-# Define RMM logging level
-target_compile_definitions(cudf PRIVATE "RMM_LOGGING_LEVEL=LIBCUDF_LOGGING_LEVEL")
+# Define logging level
+target_compile_definitions(cudf PRIVATE "CUDF_LOG_ACTIVE_LEVEL=LIBCUDF_LOGGING_LEVEL")
```

Comment:
I'm a bit confused as these are two separate env variables but seem to define the same thing? Can we combine them so we only have one env var for it?

Reply:
`LIBCUDF_LOGGING_LEVEL` is a CMake variable, while `CUDF_LOG_ACTIVE_LEVEL` is a preprocessor definition (`#define` in C/C++). They are independent entities that both need to exist since they serve different purposes. If you prefer, I can align the CMake variable's name with the underlying define, but that would be a breaking change for the build.

Reply:
I just realize that `LIBCUDF_LOGGING_LEVEL` is CMake variable but there is an env variable with the same name. So technically we have at least 3 logging variables. Probably my previous request to combine all of them into one is impossible. Maybe rename them to be something else better to understand? For example, can be like `CUDF_COMPILE_LOG_LEVEL` (so we know it is used for compiling C++ code), and `CUDF_ENV_LOG_LEVEL` so we know it is environment variable used at runtime etc.